### ISSUE TITLE
xgboost only supports int labels (0-n). This is inconsistent with other sklearn methods. But we may have to manually transform labels for classification (making sure to keep with baseestitor conventions with est.classes_ etc.)

### ISSUE THREAD

**Issue by miguelehernandez** [EpistasisLab/tpot2#45]

**Comment by miguelehernandez** [comment]
PR done, Pedro to check.
